browser/IgBlazorSamples.Test/Models/TocModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace IgBlazorSamples.Test.Models
+{
+    public class Toc
+    {
+        [JsonPropertyName("groups")]
+        public TocGroup[] Groups { get; set; }
```

Comment:
Ah, it's the built-in [`JsonSerializer` default behavior](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/how-to#serialization-behavior) (as opposed to Newtonsoft.Json). That could be switched to camelCase easily enough or in your case since I think this is about de-serialization - you can set it to [case-insensitive](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/character-casing) which is what `Http.GetFromJsonAsync` seems to be doing by default (-.-) and that's the other use of it since it's the gulp that produces it..